components/director/pkg/inputvalidation/validators_string.go

```diff
@@ -34,6 +36,20 @@ func (v *dnsNameRule) Validate(value interface{}) error {
 	return nil
 }
 
+func (v *runtimeNameRule) Validate(value interface{}) error {
```

Comment:
Should we leave the DNS compliance since right now we allow spaces in the name?

Reply:
and special symbols, and so on... I'm trying to find the place where the runtime name is "randomly generated", so we can tweak aour validation accordingly

Reply:
I wasn't able to find where the generation is happening, but I saw that in the cockpit, the name can contain underscore, which is not DNS compliant

Reply:
We should probably still have some validation for the special characters though?

Reply:
This whole function can be reduced to `validation.Field(&i.Name, validation.Required, validation.RuneLength(1, appNameLengthLimit) validation.Match(runtimeNameRgx))`

Reply:
Absolutely! Thanks @DimitarPetrov 🍻